.github/workflows/test-artifact-actions-and-stuff.yml

```diff
@@ -0,0 +1,24 @@
+name: Test artifact actions
+on: push
+
+jobs:
+  test:
+    name: Test artifact actions
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Upload
+        uses: ./.github/actions/artifact_upload
+        with:
+          name: test-artifact
+          path: test-folder/file-*.txt
+          encryption-secret: asdf
+      - name: Download
+        uses: ./.github/actions/artifact_download
+        with:
+          name: test-artifact
+          path: /tmp
+          encryption-secret: asdf
+      - name: Check existence
+        run: ls /tmp/test-folder
```

Comment:
We could not only test that the item exists here, but also that it is 
1. Not containing the original content in it's encrypted state, by downloading it through the "normal" artifact download action
2. Containing exactly the original content after decryption

Reply:
I verified both things by manually downloading the artifacts, but I can rewrite it if the test workflow is meant to be kept for merging.

Reply:
Ok, let's remove it as per Paul's comment above.

Reply:
I removed everything test related :)